lib/beaker/logger.rb

```diff
@@ -297,14 +305,18 @@ def success *args
     # @param args[Array<String>] Strings to be reported
     def notify *args
       return unless is_notify?
-      optionally_color BRIGHT_WHITE, *args
+      if @ci_build
```

Comment:
This seems like an odd way to implement this.

If we intend to be able to change colors for a `@ci_build`, maybe we should just maintain a couple of hashes with the colors, and switch the active hash based on whether we're operating in CI.


```ruby
default_colors = { :success => MAGENTA, :notify => BLUE }
ci_colors =  { :success => MAGENTA, :notify => NORMAL }

colors = @ci_build ? ci_colors : default_colors
```

Then make the method calls look like:

```ruby
def success *args
  optionally_color colors[:success], *args		
end
```